--- /tmp/SBo/VirtualBox-6.1.6/src/VBox/Frontends/VirtualBox/src/main.cpp	2020-04-09 19:41:45.000000000 +0200
+++ _modified_main.cpp	2020-04-21 20:28:07.424153681 +0200
@@ -911,6 +911,9 @@
 
 int main(int argc, char **argv, char **envp)
 {
+    /* Enable QT's SetUID setting since it is not enabled by default since QT 5.12 */
+    QCoreApplication::setSetuidAllowed(true);
+
 # ifdef VBOX_WS_X11
     /* Make sure multi-threaded environment is safe: */
     if (!MakeSureMultiThreadingIsSafe())
